<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ColorMapControl</class>
 <widget class="QGroupBox" name="ColorMapControl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>121</width>
    <height>115</height>
   </rect>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QGroupBox" name="cutLevelGroupBox">
     <property name="title">
      <string>Cut Level</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lowerCutLevelLabel">
        <property name="text">
         <string>Lo</string>
        </property>
        <property name="buddy">
         <cstring>lowerCutLevelSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="upperCutLevelLabel">
        <property name="text">
         <string>Hi</string>
        </property>
        <property name="buddy">
         <cstring>upperCutLevelSpinBox</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="2">
       <widget class="DoubleRangeSlider" name="cutLevelSlider" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="ScienceSpinBox" name="lowerCutLevelSpinBox"/>
      </item>
      <item row="2" column="1">
       <widget class="ScienceSpinBox" name="upperCutLevelSpinBox"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleRangeSlider</class>
   <extends>QWidget</extends>
   <header>doublerangeslider.h</header>
   <container>1</container>
   <slots>
    <signal>lowerValueChanged(double)</signal>
    <signal>upperValueChanged(double)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>ScienceSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>sciencespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>setLowerCutLevel(double)</slot>
  <slot>setUpperCutLevel(double)</slot>
  <slot>updateWidgets()</slot>
 </slots>
</ui>
